src/main/java/org/apache/commons/compress/archivers/dump/DumpArchiveConstants.java

```diff
@@ -135,15 +135,32 @@ public static SEGMENT_TYPE find(final int code) {
     public static final int TP_SIZE = 1024;
 
     /**
-     * NTREC value {@value}.
+     * Minimum number of kilobytes per record: {@value}.
+     */
```

Comment:
New public and protected elements need a Javadoc `@since` tag. The question really is: Does this really need to be public or can it be package-private or private?